spec/models/post_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  subject do
+    first_user = User.create(name: 'Atif', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                             bio: 'A pakistani micronaut.', posts_counter: 0)
+    Post.create(author: first_user, title: 'Hello', text: 'This is a personal post', likes_counter: 0,
+                comments_counter: 0)
+  end
+
+  before { subject.save }
+
+  it 'title cannot be blank' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'created successful' do
+    expect(subject).to be_valid
+  end
+
+  it 'title cannot be blank' do
+    subject.title = "Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+    Eius dolorum velit modi suscipit Lorem ipsum dolor sit, amet consectetur adipisicing elit. Eius dolorum
+    velit modi suscipit unde dignissimos unde dignissimos alias deserunt beatae sit
```

Comment:
- [ ] I think it would be better if you correct this above typing error about the description of your test as it has to check the `length of the title's characters` and not `the title to be blank`.
```ruby
# For example
 it "should check the number of the  title's characters" do
     ....
  end
```

Reply:
Thanks for suggesting the valuable title I will update it